### PR TITLE
Set ugettext and ungettext as alias of gettext/ngettext functions

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

**Comment by timgraham** [comment]
Does the same consideration as https://github.com/django/django/pull/7913#pullrequestreview-17806064 apply here? I'm planning to write to the mailing list about this and the handling of the other aliases [#27753](https://code.djangoproject.com/ticket/27753) today.

**Comment by claudep** [comment]
Yes and no. Frankly, the aliases here cost nothing except namespace cluttering. Considering that it's a very commonly used import, I'd rather not deprecate them. I think this is not an all or nothing decision, we might also decide to deprecate at least the less-used imports. Opening the discussion is fine.

**Comment by timgraham** [comment]
"Yes" in that any backported patches would need to be adjusted to the `u` versions (and third-party apps supporting Python 2 also must use the `u` versions), correct?

**Comment by claudep** [comment]
Correct.

**Comment by claudep** [comment]
Tim, would you like a commit with only the changes in django.utils.translation and the docs which states that gettext and ugettext are aliases? Then we could delay the deprecation (if any) and the usage in Django itself and the other docs. Maybe @aaugustin has an opinion on this, too?

**Comment by timgraham** [comment]
I'm not certain of the path of least confusion. It seems a similar situation as https://github.com/django/django/pull/7913#pullrequestreview-18699147.

**Comment by claudep** [comment]
I isolated the minimal changes I'd really like to make in Django 2.0 in #7955.

**Comment by claudep** [comment]
#7955 has been merged. What next? Close this and commit it later in the 2.0 cycle, or for the 2.1 cycle, or...?

**Comment by timgraham** [comment]
Not sure, there's hasn't been any mailing list feedback on the topic. Probably I need to raise it in a separate thread for better visibility.

File: docs/topics/i18n/translation.txt [hunk 1]
```diff
@@ -185,10 +183,10 @@ such as when the string is presented to the user.
 Pluralization
 -------------
 
-Use the function :func:`django.utils.translation.ungettext()` to specify
+Use the function :func:`django.utils.translation.ngettext()` to specify
 pluralized messages.
 
-``ungettext`` takes three arguments: the singular translation string, the plural
+``ngettext`` takes three arguments: the singular translation string, the plural
```
Comment (timgraham) [inline docs/topics/i18n/translation.txt]: ()

File: tests/flatpages_tests/test_forms.py [hunk 2]
```diff
@@ -101,4 +101,4 @@ def test_flatpage_nosites(self):
 
         self.assertEqual(
```
Comment (timgraham) [inline tests/flatpages_tests/test_forms.py]: single line looks okay here

File: tests/i18n/tests.py [hunk 3]
```diff
@@ -52,15 +52,31 @@ def patch_formats(lang, **settings):
 
 class TranslationTests(SimpleTestCase):
 
+    @translation.override('de')
+    def test_aliases(self):
+        """
+        Pre Django 2 aliases with u prefix are still available.
```
Comment (timgraham) [inline tests/i18n/tests.py]: Pre-Django 2.0

File: tests/i18n/tests.py [hunk 4]
```diff
@@ -52,15 +52,31 @@ def patch_formats(lang, **settings):
...
+        """
+        from django.utils.translation import (
```
Comment (timgraham) [inline tests/i18n/tests.py]: Why the inner import? To discourage usage in other tests?

**Pull request closed without merging**